test/e2e/defaultbackend/default_backend.go

```diff
@@ -98,4 +98,35 @@ var _ = framework.IngressNginxDescribe("Default backend", func() {
 			Expect(resp.StatusCode).Should(Equal(test.Status))
 		}
 	})
+	It("should logging access to default backend when it enabled", func() {
```

Comment:
Please change the title to something like

`enables access logging for default backend`

Reply:
Thanks, I fixed it